shared/src/business/entities/externalDocument/ExternalDocumentBase.js

```diff
@@ -0,0 +1,18 @@
+const {
+  JoiValidationConstants,
+} = require('../../../utilities/JoiValidationConstants');
+
+const externalDocumentDecorator = (obj, rawObj) => {
+  obj.category = rawObj.category;
+  obj.documentTitle = rawObj.documentTitle;
+  obj.documentType = rawObj.documentType;
+};
+
+const baseExternalDocumentValidation = {
+  category: JoiValidationConstants.STRING.required(),
+  documentTitle: JoiValidationConstants.DOCUMENT_TITLE.optional(),
+  documentType: JoiValidationConstants.STRING.required(),
+};
+
```

Comment:
While I appreciate the ♻️ reusing here, this makes me think that we can go further and create a base class that the others extend. 🤔 Nothing for now, but perhaps something for the devex backlog to consider.